staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -78,6 +78,9 @@ type Config struct {
 	// this to confirm the proxy's identity
 	ProxyClientCert []byte
 	ProxyClientKey  []byte
+
+	// Indicates if the Aggregator should send to the cluster IP (false) or route to the endpoints IP (true)
+	EnableAggregatorRouting bool
```

Comment:
I can live with this one being as-is, but when you try to wire up the ssh tunnel, that must be an interface passed through at this level and a future refactor may choose to combine them.